routes/article.js

```diff
@@ -149,10 +150,25 @@ router.post('/vote', (req, res) => {
         })
 })
 
+router.get('/', (req, res) => {
+    Article
+        .find()
+        .lean()
+        .then((art) => {
```

Comment:
we should makes this var art article because thats what we call it everywhere else(or call it art everywhere)